src/history/hs_rec.c

```diff
@@ -817,6 +824,29 @@ __hs_delete_key_from_ts_int(
     return (ret);
 }
 
+/*
+ * __wt_hs_delete_key_from_ts_btree --
+ *     Delete history store content of a given key from a timestamp.
+ */
+static int
+__wt_hs_delete_key_from_ts_btree(
```

Comment:
I feel like we could do this in less function calls as now we have:

1.  `__hs_delete_key_from_ts`
2. `__hs_delete_key_from_ts_btree`
3. `__hs_delete_key_from_ts_btree_int`

I don't think its obvious as to which you'd want to call from a caller. Perhaps merging 1 and 2 is the way to go.

Reply:
Updated as you suggested.